homeassistant/components/bluetooth_tracker/device_tracker.py

```diff
@@ -152,15 +170,15 @@ async def perform_bluetooth_update():
                     rssi = await hass.async_add_executor_job(client.request_rssi)
```

Comment:
This actually returns single byte number or `None`, it's not a tuple.

Reply:
https://github.com/microsoft/pyright/blob/db8707eb8466fa871a6b81dbdbebb42da71f80c6/packages/pyright-internal/typeshed-fallback/stdlib/struct.pyi#L9 defines return of `unpack` as  `Tuple` :thinking:

Reply:
ah, now I see your edit